src/analysis/retail/monk/mistweaver/modules/spells/Upwelling.tsx

```diff
@@ -80,13 +81,22 @@ class Upwelling extends Analyzer {
     );
   }
 
+  get averageExtraBolts() {
+    return this.extraBolts / this.castEF;
+  }
+
+  get lostEfHealing() {
+    return (this.baseEfHealing * this.averageExtraBolts) / (BASE_BOLTS + 1);
```

Comment:
For percentage of casts missed it would be 1-(avgbolts/18) since your average extra bolts per cast is effectively the percentage toward a full additional cast, not percent missed